controllers/configurationpolicy_controller.go

```diff
@@ -1599,18 +1627,16 @@ func (r *ConfigurationPolicyReconciler) determineDesiredObjects(
 	if len(desiredObjects) == 0 {
 		log.Info("Final desired object list is empty after processing selectors")
 
-		msg := fmt.Sprintf("object of kind %s has no name specified "+
-			"from the policy objectSelector nor the object metadata",
-			objGVK.Kind,
-		)
-
-		errEvent := &objectTmplEvalEvent{
-			compliant: false,
-			reason:    "objectSelector error",
-			message:   msg,
+		event := &objectTmplEvalEvent{
+			compliant: true,
+			reason:    "",
+			message: fmt.Sprintf(
+				"No objects of kind %s were matched from the policy objectSelector nor the object metadata",
```

Comment:
It's a good idea, but I think I could go either way on adding "skipped" to the message. On the one hand, it would be more informative and would raise the feature to those that weren't aware of it. On the other hand, it could be confusing if users aren't aware skipping was a thing. Maybe the message could be even more targeted as ", or they were skipped by the `skipObject` template function"?

Reply:
@JustinKuli and @dhaiducek in the latest update, I made the compliance message specific based on if `skipObject` was called or if an `objectSelector` is used. Let me know what you think.